docs/src/modules/components/DemoEditor.tsx

```diff
@@ -51,9 +51,11 @@ const StyledSimpleCodeEditor = styled(SimpleCodeEditor)(({ theme }) => ({
   '& textarea': {
     outline: 0,
   },
-  '& > textarea, & > pre': {
-    // Override inline-style
-    whiteSpace: 'pre !important',
+  '& [class^=language]': {
```

Comment:
I found that only the styles on the inner `code` element matters, the `textarea` and `pre` doesn't matter